cloud/jenkins/ps_operator_eks_version.groovy

```diff
@@ -246,6 +319,7 @@ pipeline {
                         ./cloud/local/checkout $GIT_REPO $GIT_BRANCH
 
                         cp $CLOUD_SECRET_FILE ./source/e2e-tests/conf/cloud-secret.yml
+                        chmod 600 ./source/e2e-tests/conf/cloud-secret.yml
```

Comment:
We will need to use stash/unstash to prepare sources after prepareNode on each node. Take a look at GKE pipeline, because it had parallel from before.